meta-ostro/classes/supported-recipes.bbclass

```diff
@@ -270,8 +364,14 @@ python supported_recipes_eventhandler() {
             collections = supported_recipes.find_collections(pn)
             return ' (would be supported in %s)' % ' '.join(collections) if collections else ''
 
+        def re_escape(str):
+            # In contrast to re.escape(), we only escape + (as in gtk+3).
+            # Escaping all non-alphanumerics makes many entries (like linux-yocto)
+            # unnecessarily less readable (linux\-yocto).
+            return ''.join(['\\+' if x == '+' else x for x in str])
+
```

Comment:
I can use re.sub(), but I would still do it in a utility function, to avoid cut-and-paste. Agreed, or would you prefer to have the same expression repeated, once for pn, once for collection?